policyengine_us/variables/gov/states/md/tax/income/agi/subtractions/md_total_subtractions.py

```diff
@@ -9,4 +9,8 @@ class md_total_subtractions(Variable):
     definition_period = YEAR
     defined_for = StateCode.MD
 
-    formula = sum_of_variables(["md_dependent_care_subtraction"])
+    def formula(tax_unit, period, parameters):
```

Comment:
```suggestion
    formula = sum_of_variables("gov.states.md.tax.income.agi.subtractions.sources")
```

Reply:
But how does `sum_of_variables` know what the period is?

Reply:
It assumes the period is the current period.
```
sum_of_variables("tax.taxes")
```
is a shortcut for:
```
def formula(entity, period, parameters):
  return sum([entity(variable, period) for variable in parameters(period).tax.taxes])
```